sechub-server/src/main/resources/db/migration/U6__provide_metadata_in_product_result2.sql

```diff
@@ -0,0 +1,3 @@
+DROP TABLE scan_product_result
+;
```

Comment:
Absolute wrong!

 we must undo ALTER TABLE scan_product_result
  ADD COLUMN meta_data text -- accept long text, also null accepted
;